js/main.js

```diff
@@ -65,45 +67,32 @@ function loadView(event) {
   } else if (data.view === 'entries') {
     $getForm.className = 'hidden';
     $getEntries.className = 'get-entries';
-    hideFormShowEntries();
   }
 
   for (var i = data.entries.length - 1; i >= 0; i--) {
     ulEl.prepend(renderList(data.entries[i]));
   }
 }
 
-var viewEntries = document.querySelector('.entry-view');
+var $view = document.querySelectorAll('.view');
+
 var $getEntries = document.querySelector('.get-entries');
 var $getForm = document.querySelector('.get-form');
-viewEntries.addEventListener('click', hideFormShowEntries);
-
-function hideFormShowEntries(event) {
-  $getForm.className = 'hidden';
-  $getEntries.className = 'get-entries';
-  data.view = 'entries';
-}
-
-var $newEntry = document.querySelector('.new-entry');
-$newEntry.addEventListener('click', showFormHideEntries);
-
-function showFormHideEntries(event) {
-  $getForm.className = 'get-entries';
-  $getEntries.className = 'hidden';
-  data.view = 'entry-form';
-
-}
-
-/* var $save = document.querySelector('.save');
-$save.addEventListener('click', saveEntry);
-function saveEntry(event) {
-  if (data.entries.length === 0) {
-    var $noDiv = document.querySelector('.no-entries');
-    var $noEntriesMsg = document.createElement('p');
-    $noEntriesMsg.textContent = 'No entries have been recorded.';
-    $noDiv.appendChild($noEntriesMsg);
+var $body = document.querySelector('body');
+
+$body.addEventListener('click', toggleView);
```

Comment:
Instead of putting this on the `body` element, I would apply it directly to each `a` element that is used for navigating to a different view